Gemfile

```diff
@@ -1,22 +1,23 @@
 source "https://rubygems.org"
 
-gem "refinerycms-authentication-devise", '~> 1.0.4'
+gem "refinerycms-authentication-devise", '~> 2.0'
 
 gemspec
 
-gem 'refinerycms', '~> 3.0.5'
+gem 'refinerycms', '~> 4.0'
+
 
 group :development, :test do
-  gem 'refinerycms-testing', '~> 3.0.5'
+  gem 'refinerycms-testing', '~> 4.0'
 end
 
 # Add the default visual editor, for now.
 gem 'refinerycms-wymeditor', ['~> 1.0', '>= 1.0.6']
```

Comment:
You should use this version of refinerycms-wymeditor : `gem 'refinerycms-wymeditor', ['~> 2.0', '>= 2.0.0']`